examples/js/loaders/GLTFLoader.js

```diff
@@ -261,7 +262,14 @@ THREE.GLTFLoader = ( function () {
 
 				case 'spot':
 					lightNode = new THREE.SpotLight( color );
-					lightNode.position.set( 0, 0, 1 );
+					// Handle spotlight properties.
+					var spot = light.spot || {};
+					var innerConeAngle = spot.innerConeAngle !== undefined ? spot.innerConeAngle : 0;
+					var outerConeAngle = spot.outerConeAngle !== undefined ? spot.outerConeAngle : Math.PI / 4.0;
+					lightNode.angle = outerConeAngle;
+					lightNode.penumbra = 1.0 - innerConeAngle / outerConeAngle;
+					lightNode.target.position.set( 0, 0, 1 );
```

Comment:
The target of a spot is `0, 0, 1` by default?

Reply:
The local direction of the glTF light is always +Z; the only way to rotate it is by rotating the object or its parents in the scene graph.